public/api-merged.json

```diff
@@ -6469,6 +6469,145 @@
         }
       }
     },
+    "/provisioning/contact-points": {
```

Comment:
I think this is an artefact of https://github.com/grafana/grafana/pull/47197 but `api-merged` is a new file to me.